README.md

```diff
@@ -126,6 +127,54 @@ client = Redis.new(semian: {
 })
 ```
 
+#### Net::HTTP
+For the `Net::HTTP` specific Semian adapter, since many external libraries may create
+HTTP connections on the user's behalf, the parameters are instead provided
+by calling specific functions in `Semian::NetHTTP`, perhaps in an initialization file.
+
+To give Semian parameters, assign a `proc` to `Semian::NetHTTP.semian_configuration`
+that takes two parameters, `host` and `port like `127.0.0.1` and `80` or `github_com` and `80`,
+and returns two values, a `semian_identifier` and a `Hash` with the keys as follows.
+
+```ruby
+SEMIAN_OPTIONS = { tickets: 1,
+                   success_threshold: 1,
+                   error_threshold: 3,
+                   error_timeout: 10 }
+Semian::NetHTTP.semian_configuration = proc do |host, port|
+  semian_identifier = "nethttp_#{host}_#{port}"
+  # Let's make it only active for github.com
+  if semian_identifier == "nethttp_github.com_80"
+    [semian_identifier, SEMIAN_OPTIONS]
+  else
+    [semian_identifier, nil]
+  end
+end
+
+# Called from within API:
+# semian_identifier, semian_options = Semian::NetHTTP.semian_configuration("github.com", 80)
+```
+
+The `semian_identifier` used as the name of the resource, and the `semian_options` are the
+configuration parameters. A return value of `nil` means Semian is disabled for that
+`Net::HTTP` endpoint. This works well since the result of a failed Hash lookup is `nil` also.
+As such, this particular adapter defaults to whitelisting, although the
+behavior can be changed to blacklisting or even be completely disabled by varying
+the use of returning `nil`.
+
+Since we envision this particular adapter can be used in combination with many
+external libraries, we added functionality to expand the Exceptions that can be
+tracked as part of Semian's circuit breaker. Add exceptions and reset to the
+[`default`][nethttp-default-errors] list using the following:
```

Comment:
You should give an example of why this is useful, e.g. a library might re-raise Net::HTTP exceptions already and then you need this.